cli-plugins/manager/hooks_test.go

```diff
@@ -108,3 +109,34 @@ func TestPluginMatch(t *testing.T) {
 		assert.Equal(t, match, tc.expectedMatch)
 	}
 }
+
+func TestAppendNextSteps(t *testing.T) {
+	testCases := []struct {
+		processed   []string
+		expectedOut []string
+	}{
+		{
+			processed:   []string{},
+			expectedOut: []string{},
+		},
+		{
+			processed:   []string{"", ""},
+			expectedOut: []string{},
+		},
+		{
+			processed:   []string{"Some hint", "", "Some other hint"},
+			expectedOut: []string{"Some hint", "", "Some other hint"},
```

Comment:
Are we sure we want to include empty lines if the plugin returns them?

Reply:
Not sure either, but this was the behavior we had so didn't want to change anything other than just handling the empty message.

Reply:
fair enough for now

Reply:
Tbh, I would be okay with removing empty lines (actually that was the first thing I did, but then decided not to change the behavior). I can do that in this PR if we want to, @laurazard  any opinions?